README.md

```diff
@@ -60,7 +59,241 @@ To support these changes, there have been a number of changes to the SubnetEVM b
 
 ## Create an EVM Subnet on a Local Network
 
-See [this tutorial](https://docs.avax.network/subnets/create-a-local-subnet).
+### Clone Subnet-evm
+
+First install Go 1.17.9 or later, however as the time of writing, please don't use Go v1.18.x versions. Follow the instructions [here](https://golang.org/doc/install). You can verify by runing `go version`.
+
+Set `$GOPATH` environment variable properly for Go to look for Go Workspaces. Please read [this](https://go.dev/doc/gopath_code) for details. You can verify by running `echo $GOPATH`.
+
+As a few software will be installed into `$GOPATH/bin`, please make sure that `$GOPATH/bin` is in your `$PATH`, otherwise, you may get error running the commands below.
+
+Download the `subnet-evm` repository into your `$GOPATH`:
+
+```sh
+cd $GOPATH
+mkdir -p src/github.com/ava-labs
+cd src/github.com/ava-labs
+git clone git@github.com:ava-labs/subnet-evm.git
+cd subnet-evm
+```
+
+This will clone and checkout to `master` branch.
+
+### Run Local Network
+
+[`scripts/run.sh`](https://github.com/ava-labs/subnet-evm/blob/master/scripts/run.sh) automatically installs `avalanchego`, sets up a local network,
+and creates a `subnet-evm` genesis file. The usage of this script is
+
+```bash
+./scripts/run.sh [AVALANCHEGO VERSION] [GENESIS_ADDRESS]
+```
+
+```bash
+# to startup a local cluster (good for development)
+cd ${HOME}/go/src/github.com/ava-labs/subnet-evm
+git pull
+./scripts/run.sh 1.7.11 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
```

Comment:
optional: bump version since we are here?

Reply:
Addressed